tst/CTA.Rules.Test/TempRules/microsoft.owin.security.json

```diff
@@ -902,6 +902,198 @@
           ]
         }
       ]
+    },
+    {
```

Comment:
where are the projects to test each rule ?

Reply:
Inside of the TestProjects repo. These rules specifically are covered by the AspNetRoutes.sln solution.